googlecloud-plugins/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2015-2016 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+
+  <parent>
+    <artifactId>hydrator-plugins</artifactId>
+    <groupId>co.cask.hydrator</groupId>
+    <version>1.5.0-SNAPSHOT</version>
+  </parent>
+
+  <properties>
+    <!-- properties for script build step that creates the config files for the artifacts -->
+    <app.parents>system:cdap-etl-batch[4.0.0-SNAPSHOT,4.1.0-SNAPSHOT),system:cdap-data-pipeline[4.0.0-SNAPSHOT,4.1.0-SNAPSHOT),system:cdap-data-streams[4.0.0-SNAPSHOT,4.1.0-SNAPSHOT)</app.parents>
+  </properties>
+
+  <name>Googlecloud Plugins</name>
+  <artifactId>googlecloud-plugins</artifactId>
+  <modelVersion>4.0.0</modelVersion>
+
+  <dependencies>
+    <dependency>
+      <groupId>co.cask.hydrator</groupId>
+      <artifactId>core-plugins</artifactId>
```

Comment:
you need to refactor things then. Including core-plugins as a compile dependency means that jar will get bundled into this one, which is bad. Can you just have this plugin in core? Otherwise you need to move things into a common module that they both use.